R/Data-methods.R

```diff
@@ -380,7 +380,7 @@ setMethod(
 #'
 #' @return The new, updated [`DataOrdinal`] object.
 #'
-#' @aliases update-DataOrdinal
```

Comment:
I think this should have worked (since we have other examples such as in https://github.com/Roche/crmPack/blob/main/_pkgdown.yaml#L258), maybe just need to run Build > Document in RStudio to populate the .rd file accordingly?